pkg/mpc-examples/package.json

```diff
@@ -26,6 +26,7 @@
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.2.1",
     "@nomiclabs/hardhat-waffle": "^2.0.3",
+    "@openzeppelin/contracts": "3.4.2-solc-0.7",
```

Comment:
I don't see anything from OZ being used here. Is this preemptive?

Reply:
Good catch -- I had initially made this Ownable, but then scrapped that for simplicity.

I'm torn on removing the dependency though since this will be a base from which we/others will build other sample controllers. I completely agree it's useless for this controller, but it's still handy for downstream controllers.

Reply:
Also if we make the factory `Ownable`, we would keep this. See comment below

Reply:
now using Ownable